autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/BasicSelect/BasicSelect.tsx

```diff
@@ -20,6 +23,10 @@ const BasicSelect: React.FC<BasicSelectProps> = ({ id }) => {
   const hasAsynchronous = useSelector<RootState, boolean>(
     (state) => state.courseCards[id].hasAsynchronous || false,
   );
+  const includeFull = useSelector<RootState, boolean>(
+    (state) => state.courseCards[id].includeFull || false,
+  );
+  const dispatch = useDispatch();
 
   // shows placeholder text if no course is selected
```

Comment:
I noticed that if a course doesn't have online or honors sections, the placeholder text will show up and disallow users from using the include full checkbox (ex: JAPN 101 in fall 2020). This should be changed so it's always visible, but I'm not sure if it would be more intuitive to leave the placeholder text below it or just remove it entirely. For now I'd leave it and we could discuss what we think in the next meeting